src/CqlBuilderPanel/codesSection/codesSubSection/savedCodesSubSection/SavedCodesSubSection.tsx

```diff
@@ -357,119 +357,112 @@ export default function SavedCodesSubSection({
 
   return (
     <div>
-      <ExpandingSection
```

Comment:
this is deleted. should the import be deleted as well if not being used?

Reply:
@adongare Will do, thanks!